README.md

```diff
@@ -86,6 +86,7 @@ Tier 3:
   * x86_64-unknown-linux-gnux32
   * x86_64-unknown-openbsd
   * x86_64-unknown-redox
+  * loongarch64-unknown-linux-gnuf64
```

Comment:
We're now *Tier 2 with host tools*, so please move the (corrected) tuple here to the appropriate location too.